src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.portal.portlet
+
+import spock.lang.Specification
+
+/**
+ * <short description>
+ *
+ * <detailed description>
+ *
+ * @since: 1.0.0
+ * @author: Tobias Koch
+ */
+class OMEROClientPortletSpec extends Specification {
+
+
+    def "GetPortletContent"() {
+    }
+
```

Comment:
How do you want to test that?
imo it is sufficient to just create the class, no need to create empty test classes otherwise they might stay empty

Reply:
You are right. I let my IDE create the tests.
```suggestion
```
You can go ahead and commit this suggestion so we both get credit.

Reply:
Thanks!